frontend/app/adapters/folder.js

```diff
@@ -13,7 +13,23 @@ export default ApplicationAdapter.extend({
     return super.urlForQuery(query, modelName);
   },
 
+
+  /* eslint-disable no-unused-vars */
+  urlForFindRecord(id, modelName, snapshot) {
+    return `/api/${this.pathForType()}/${id}`;
+  },
+
+  urlForCreateRecord(modelName, snapshot) {
+    return `/api/${this.pathForType()}`;
+  },
+
+  urlForUpdateRecord(id, modelName, snapshot) {
+    return `/api/${this.pathForType()}/${id}`;
+  },
+  /* eslint-enable no-unused-vars */
```

Comment:
Not very proud of this. Can probably be summarized in a buildURL function or so that is provided...

Reply:
do you need an adapter for this ? looks pretty standard to me ?!